frontend/app/src/composables/history/filter-paginate.ts

```diff
@@ -16,7 +18,7 @@ interface FilterSchema<F, M> {
   filters: Ref<F>;
   matchers: ComputedRef<M[]>;
 
-  updateFilter(filter: F): void;
+  updateFilter: (filter: F) => void;
 
   RouteFilterSchema: ZodSchema;
```

Comment:
Hmm here and in the empty one this is not marked as optional but when called in the composable it is.

Reply:
`RouteFilterSchema` right? 

yeah, I'll remove the null check since in every instance it will always be there...
I initially made them optional.

Reply:
done